airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -318,18 +318,18 @@ public List<CheckedConsumer<JdbcDatabase, Exception>> getCheckOperations(final J
         PostgresUtils.checkFirstRecordWaitTime(config);
       });
 
-      // Verify that the db user has required privilege to perform replication.
+      // Verify that the db user has required privilege to perform replication. The user has to be super user of to has replication access.
```

Comment:
super user of db?

Reply:
My bad .. I will modify the comment.
Basically the db user needs to be superuser or to have replication privilege.